packages/esm-patient-common-lib/src/get-patient-uuid-from-url.ts

```diff
@@ -1,4 +1,7 @@
+import { getPatientChartStore } from './store/patient-chart-store';
+
 export function getPatientUuidFromUrl(): string {
   const match = /\/patient\/([a-zA-Z0-9\-]+)\/?/.exec(location.pathname);
-  return match && match[1];
+  const patientUuidFromUrl = match && match[1];
+  return patientUuidFromUrl || getPatientChartStore().getState().patientUuid;
```

Comment:
The function name `getPatientUuidFromUrl` suggests it only retrieves the UUID from the URL, but it actually has a fallback behavior that gets the UUID from the store state. I suggest either:

1. Rename the function to better reflect its complete behavior, e.g., `getActivePatientUuid` or `resolvePatientUuid`, since it resolves the patient UUID from either the URL or store
2. OR split this into two separate concerns if these use cases need to be handled differently

This would make the function's behavior more predictable from its name

Reply:
Yes @usamaidrsk!
I agree with you on this, and am seeking @denniskigen's advice too. But do note that this is an API in the Patient Common Lib, and changing the function will require updating all the places this function is being used, and hence will become a breaking change, since this will also affect other implementations to update their code when released.

One thing is, we can deprecate this function and in the message we can ask them to update the API in their respective code. This way, it'll not be a breaking change.
Thanks!